docs/build/manifest.md

```diff
@@ -101,6 +101,56 @@ dataSources:
               - Transfer(address indexed from, address indexed to, uint256 amount)
 ````
 
+  </CodeGroupItem>
+  <CodeGroupItem title="v1.0.0 Cosmos">
+
+``` yml
+specVersion: 1.0.0
+name: cosmos-juno-subql-starter
+version: 0.0.1
+runner:
+  node:
+    name: '@subql/node-cosmos'
+    version: latest
+  query:
+    name: '@subql/query'
+    version: latest
+description: 'This project can be use as a starting point for developing your Cosmos (CosmWasm) based SubQuery project using an example from Juno'
+repository: https://github.com/subquery/cosmos-subql-starter
+schema:
+  file: ./schema.graphql
+network:
+  chainId: juno-1
+  endpoint: https://rpc.juno-1.api.onfinality.io
+  # Optionally provide the HTTP endpoint of a full chain dictionary to speed up processing
+  dictionary: https://api.subquery.network/sq/subquery/cosmos-juno-1-dictionary
+dataSources:
+  - kind: cosmos/Runtime
+    startBlock: 1
+    chainTypes:
+      cosmos.slashing.v1beta1:
+        file: "./proto/cosmos/slashing/v1beta1/tx.proto"
+        messages:
+         - "MsgUnjail"
```

Comment:
This is the only undocumented part for cosmos. Similar to with substrate where chain types are loaded, we can load protobuf messages specific to cosmos chains. If most are just using Wasm this should be already included. It hasn't been thoroughly tested as its hard to find a usecase. 

I think we can add this in another PR.